package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "idp-profile-ui",
-  "version": "7.2.0",
+  "version": "7.2.1",
```

Comment:
Is it possible to update this automatically? Something like [this](https://github.com/silinternational/idp-id-broker/blob/c46958cd989b40811c83949bfbd7362f0dcbdf2c/application/common/config/main.php#L206).

Reply:
Yeah, every time I do work here I have forgotten the syntax and the ide plugin isn't as helpful as it is for Svelte.

Reply:
Do you mean actually changing the package.json file or just using the github.ref_name?

Reply:
Either. I don't know where this is used, other than the `installed-packages.json` file.

Reply:
It's also printed to the console: Environment: appsdev-pw.iidp.net Release: 7.1.1 DSN: ...

Reply:
I'm not sure how we would automate at it at this point but if it's helpful we could do away with it. Since we don't use a github workflow to deploy it (at least I think) we can't exactly use one to set an env.